ocs_ci/deployment/baremetal.py

```diff
@@ -706,6 +706,16 @@ def clean_disk():
             logger.info(out)
 
     for worker in workers:
+        cmd = """lsblk --all --noheadings --output "KNAME,PKNAME,TYPE,MOUNTPOINT" --json"""
+        out = ocp_obj.exec_oc_debug_cmd(node=worker.name, cmd_list=[cmd])
+        disk_to_ignore_cleanup_raw = json.loads(str(out))
+        disk_to_ignore_cleanup_json = disk_to_ignore_cleanup_raw["blockdevices"]
+        for disk_to_ignore_cleanup in disk_to_ignore_cleanup_json:
+            if disk_to_ignore_cleanup["mountpoint"] == "/boot":
+                logger.info(
+                    f"Ignorning disk {disk_to_ignore_cleanup['pkname']} for cleanup"
```

Comment:
maybe mention in the log message the reason why it's not being cleaned

Reply:
Done added the reason for skipping